app/views/container_topology/show.html

```diff
@@ -0,0 +1,155 @@
+<!doctype html>
+<html class="no-js">
+<head>
+  <meta charset="utf-8">
+
+  <meta name="viewport" content="width=device-width">
+  <style>
+    /*@font-face {*/
+      /*font-family: 'FontAwesome';*/
+      /*src: url('../assets/fonts/fontawesome-webfont.svg') format('svg');*/
+    /*}*/
+    * {
+      box-sizing: border-box;
+      font-family: sans-serif;
+      font-size: 15px;
+    }
+    body {
+      margin: 0px;
+    }
+    kubernetes-topology-graph {
+      border: 10px solid lightgray;
+      position: fixed;
+      height: 76%;
+      width: 98%;
+      padding: 10px;
+    }
+    .legend {
+      display: inline-block;
+      padding: 0 10px;
+      font-family: "FontAwesome";
+
+    }
+    .legend label {
+      font-weight: 400;
+    }
+
+    .refresh {
+      display: inline-block;
+      width:35px;
+      height:35px;
+      padding:0;
+    }
+
+    #selected {
+      float: right;
+      display: block;
+      margin-top: 15px;
+    }
+
+    kubernetes-topology-icon {
```

Comment:
> There was a thought that this widget was generic and could be used for, say, vmware topology. In its present form it can't. I think the question is whether or not this has to be forked in order to make it more generic.

@Fryguy
OK now I see what you meant by addressing the naming.
The initial requirement for this was to be for containers provider, and the component is currently used in 2 other projects focused on containers management areas.
Lets see what @stefwalter thinks on generalizing.
I'd really recommend to avoid forking. Who is supposed to maintain the sync of the fork with the original component?

cc @simon3z @itamarh